📝 Clarify automated manual backups (#2721)
platformsh/platformsh-docs: docs/src/environments/backup.md

{{< /codetabs >}}

### Automate manual backups

You can also automate the process of creating manual backups through [cron jobs](../create-apps/app-reference.md#crons).

> The cron job should use the CLI command to backup the environment.

Reviewer: 🛠️ To avoid the use of `should` and fix the typo?

```suggestion
In this case, the cron job uses the CLI command to back up the environment.
```

Author: I don't think `in this case` works ideally here, because it's not clarifying cron jobs in general to this specific example. How about without that start:
```suggestion
The cron job uses the CLI command to back up the environment.
```
